### PR TITLE
Added outline for docs site.

### PR DESCRIPTION
**Pull request by ssk2**
Moved the docs site to use a similar format as Marathon (https://mesosphere.github.io/marathon).  The docs are mostly the same as what was in the README and docs folder before, just merged and cleaned up a little. 

I've also added a small set of instructions on how to install from packages.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Chronos Docs and Website
+
```
Comment (elingg) [inline docs/README.md]: I would rename this "Chronos Documentation Website" to be clear

File: docs/docs/api.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
+---
+title: REST API
+---
+
+# REST API
+
+You can communicate with Chronos using a RESTful [JSON][] API over HTTP.
+Chronos nodes usually listen on `port 8080` for API requests.
+All examples in this section assume that you've found a running leader at `chronos-node.airbnb.com:8080`.
+
```
Comment (elingg) [inline docs/docs/api.md]: Not you, but can we brand this as chronos-node.mesosphere.com:4400. Let's change ports from 8080 to 4400 as the standard since 8080 is typically used by Marathon.

File: docs/docs/api.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+When you have multiple Chronos nodes running, only one of them will be elected as the leader.
+The leader is the only node that responds to API requests, but if you attempt to talk to a non-leader your request will automatically be redirected to a leader.
+
+### Listing Jobs
+
+* Endpoint: __/scheduler/jobs__
+* Method: __GET__
+* Example: `curl -L -X GET chronos-node:8080/scheduler/jobs`
+* Response: JSON data
+
```
Comment (elingg) [inline docs/docs/api.md]: Port 4400?

File: docs/docs/debugging.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    Task with id 'value: TASK_ID **FINISHED**
+
+To find debug logs on the mesos slave, look in `/tmp/mesos/slaves` on the slave instance (unless you've specifically supplied a different log folder for mesos). For example:
+
+    /tmp/mesos/slaves/
+
+In that dir, the current slave run is timestamped so look for the most recent.
+Under that is a list of frameworks; you're interested in the Chronos framework.
+For example:
+
```
Comment (elingg) [inline docs/docs/debugging.md]: can we make this <timestamp> instead of STAMP

File: docs/docs/faq.md [hunk 5]
```diff
@@ -1,14 +1,37 @@
...
+If you use the cURL command line tool, you can use the `-L` flag and hit any Chronos node and you will get a
+307 REDIRECT to the leader.
+
+
+### <a name="chronos-zookeeper"></a>How does Chronos use ZooKeeper?
+
+Chronos registers itself with [ZooKeeper][ZooKeeper] at the location `/chronos/state`. This value can be changed via the configuration file.
+
+### <a name="chronos-cassandra"></a>How does Chronos use Cassandra?
+
```
Comment (elingg) [inline docs/docs/faq.md]: I would say "job history, reporting, and statistics" here

File: docs/docs/index.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+### Install Dependencies
+
+#### Debian Linux:
+    sudo apt-get install autoconf make gcc cpp patch python-dev git libtool default-jdk default-jdk-builddep default-jre gzip libghc-zlib-dev libcurl4-openssl-dev
+
+#### Fedora Linux:
+    sudo yum install autoconf make gcc gcc-c++ patch python-devel git libtool java-1.7.0-openjdk-devel zlib-devel libcurl-devel openssl-devel cyrus-sasl-devel
+
+Make sure you're using Java 7: `sudo alternatives --config java`
+
```
Comment (elingg) [inline docs/docs/index.md]: Let's have the option here of only installing Mesos and not building it. Most users will want to simply install it.

File: docs/docs/index.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    git clone https://github.com/apache/mesos.git
+    cd mesos/
+    git checkout 
+    export JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk-1.7.0.45.x86_64/
+    ./bootstrap
+    ./configure --with-webui --with-included-zookeeper --disable-perftools --enable-frame-pointers
+    make
+    sudo make install
+
```
Comment (elingg) [inline docs/docs/index.md]: Let's have the option here of only installing Chronos and not building it. Most users will want to simply install it using yum install, for example.

File: docs/docs/api.md (hunk 2 above)
Comment (elingg) [inline docs/docs/api.md]: If we do update the port, we should also update the default configuration. Maybe this should be a separate PR, however. This PR is already pretty big.

File: docs/docs/faq.md (hunk 5 above)
Comment (elingg) [inline docs/docs/faq.md]: Also, we should probably mention that Cassandra is optional.

File: docs/docs/webui.md [hunk 8]
```diff
@@ -19,7 +20,7 @@ There are also two included example YAML files that make use of this configurati
 * [config/local_scheduler.yml](/mesos/chronos/blob/master/config/local_scheduler.yml#L4)
 * [config/local_scheduler_nozk.yml](/mesos/chronos/blob/master/config/local_scheduler_nozk.yml#L7)
 
-## Build Requirements
+### Build Requirements
```
Comment (elingg) [inline docs/docs/webui.md]: It seems that the issue is closed (#42) some time ago so I wouldn't mention it.  @mlunoe, did you encounter any out of memory issue when you built the Chronos UI? Just want to double check.

File: docs/support.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+# Getting Support for Chronos
+
+For questions and discussions around Chronos, please use the Google Group "chronos-scheduler":
+[Chronos Scheduler Group](https://groups.google.com/forum/#!forum/chronos-scheduler).
+
+Also join us on IRC in #mesos on freenode.
+
+## Reporting Bugs
+
```
Comment (elingg) [inline docs/support.md]: Nice bug reporting/debugging tips!

File: docs/docs/webui.md (hunk 8 above)
Comment (mlunoe) [inline docs/docs/webui.md]: So I experienced errors when I tried to compile the Chronos UI, specifically I got errors when in the minification step, some time back on my own machine (16 GB Memory). I think we can keep this in the documentation for now. And then we will fix these problems, when will start to rebuild the UI.

File: docs/resources.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
+---
+layout: narrow
+tab: resources
+title: Chronos Resources
+---
+
```
Comment (elingg) [inline docs/resources.md]: Nice links!

**Comment by elingg** [comment]
Looks awesome, @ssk2! I would like to look it over together and if you could also address my comments, that would be great. After that, I can do a final review.

File: docs/docs/webui.md (hunk 8 above)
Comment (elingg) [inline docs/docs/webui.md]: Makes sense, we can add a disclaimer saying, "When building and optimizing the assets, make sure you have at least 1GB of RAM available" without linking to the issue.

File: docs/docs/api.md (hunk 2 above)
Comment (ssk2) [inline docs/docs/api.md]: @elingg - what do you mean by the default configuration?

Comment (ssk2) [inline docs/docs/api.md]: Also, trying to steer clear of Mesosphere mentions since this repo is technically owned by the Mesos organisation.

Comment (elingg) [inline docs/docs/api.md]: Cool, we discussed this separately. It's using Chaos and ScallopConf which gives us by default 8080. About steering clear of Mesosphere mentions, that makes sense. If you check the license file though there is ownership by Airbnb and Mesosphere.

**Comment by elingg** [comment]
:+1: LGTM, feel free to merge when ready as long as the build passes.
